.travis.yml

```diff
@@ -1,7 +1,7 @@
 os:
   - osx
-osx_image: xcode9
+osx_image: xcode10.2
 language: swift
 script:
-- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -sdk iphonesimulator11.0 -destination "OS=11.0,name=iPhone 6s" | xcpretty
-- xcodebuild -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit_ExampleUITests -sdk iphonesimulator11.0 -destination "OS=11.0,name=iPhone 6s" test -enableCodeCoverage YES
+- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -sdk iphonesimulator12.2 -destination "OS=12.2,name=iPhone Xs" | xcpretty
+- xcodebuild -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit_ExampleUITests -sdk iphonesimulator12.2 -destination "OS=12.2,name=iPhone Xs" test -enableCodeCoverage YES
```

Comment:
do we definitely want to use a 12.2 sim here? Not sure whether changing the sim and ios version will impact the ui tests in the project

Reply:
Hmm I ran the tests locally and there were no issues.

Reply:
cool, no worries then!